tests/constructor/test_constructor_with_conf_structure.py

```diff
@@ -0,0 +1,228 @@
+import sys
```

Comment:
I see you pass the same structure with which `Constructor` was already tested in the `tests/constructor/test_constructor.py`. I think we don't need these tests, because config structure and constructor should be tested independently